### PR TITLE
AlphaFilter add constructor that takes alpha and use in EKF

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: AlphaFilter/AlphaFilter.hpp [hunk 1]
```diff
@@ -85,7 +87,11 @@ class AlphaFilter {
 	 *
 	 * @return retrieve the filtered result
 	 */
-	void update(const T &sample) { _filter_state = updateCalculation(sample); }
+	const T& update(const T &sample)
```
Comment (MaEtUgR) [inline AlphaFilter/AlphaFilter.hpp:90]:
```suggestion
	const T &update(const T &sample)
```
or adapt the getter just below depending on the desired style, I have not hard feelings on this one.

**Review by MaEtUgR — DISMISSED** [review]
Great! I should have done that from the beginning.

File: AlphaFilter/AlphaFilter.hpp (hunk 1 above)
Comment (MaEtUgR) [reply AlphaFilter/AlphaFilter.hpp:90]: We're doing it the other way for parameters:
e.g. `void setVelocityGains(const matrix::Vector3f &P, const matrix::Vector3f &I, const matrix::Vector3f &D);`
Or is that just me going against the stream?
No also here `Subscription &operator=(const Subscription &other)` 🤷‍♂️

**Comment by dagar** [comment]
Formatted with PX4/Firmware astyle.

File: AlphaFilter/AlphaFilter.hpp (hunk 1 above)
Comment (dagar) [reply AlphaFilter/AlphaFilter.hpp]: I'm happy with whatever can be done automatically (astyle or clang-format).

**Pull request merged**